crates/ads126x/src/register.rs

```diff
@@ -0,0 +1,94 @@
+use bitflags::bitflags;
+
+#[repr(u8)]
+pub enum Register {
+    ID = 0x00,
+    POWER = 0x01,
+    INTERFACE = 0x02,
+    MODE0 = 0x03,
+    MODE1 = 0x04,
+    MODE2 = 0x05,
+    INPMUX = 0x06,
+    OFCAL0 = 0x07,
+    OFCAL1 = 0x08,
+    OFCAL2 = 0x09,
+    FSCAL0 = 0x0A,
+    FSCAL1 = 0x0B,
+    FSCAL2 = 0x0C,
+    IDACMUX = 0x0D,
+    IDACMAG = 0x0E,
+    REFMUX = 0x0F,
+    TDACP = 0x10,
+    TDACN = 0x11,
+    GPIOCON = 0x12,
+    GPIODIR = 0x13,
+    GPIODAT = 0x14,
+    ADC2CFG = 0x15,
+    ADC2MUX = 0x16,
+    ADC2OFC0 = 0x17,
+    ADC2OFC1 = 0x18,
+    ADC2FSC0 = 0x19,
+    ADC2FSC1 = 0x1A,
+}
+
+bitflags! {
+    pub struct IdRegister: u8 {
+        const _ = !0; // Source may set any bits
```

Comment:
Does documentation like this work to indicate that the bitflags may have specific (in this case any) bits changed and they should be accounted for? For coupled bits that represent a number or different cases, we cannot express them as bitflags constants.

Reply:
You could add a doc comment so it's more visible for people reading the docs. So,
/// doc comment 
pub struct IdRegister....

Reply:
I didn't mean doc comment wise. The bitflags macro's documentation states that you should make sure any bits that can be assigned to have defined flags. When you don't want them named or accessible via `.contains()` the docs say to assign those bits to _ as I've done here.

Is the inline comment clear enough to you that all the bits may be set by the ADC and should therefore be accounted for by the bitflags macro?

Reply:
I see, yeah that in-line comment makes sense.